subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -8,6 +8,7 @@
 #include <limits.h>
 #include <assert.h>
 #include <string.h>
+#include <toolchain/common.h>
```

Comment:
`<zephyr/toolchain/common.h>`

Reply:
I've switched this to <zephyr/toolchain.h> as recommended by @stephanosio